packages/statistics/index.rst

```diff
@@ -314,40 +314,62 @@ use the :mod:`scipy.stats` sub-module of `SciPy
 Student's t-test: the simplest statistical test
 ------------------------------------------------
 
-1-sample t-test: testing the value of a population mean
+One-sample tests: testing the value of a population mean
 ........................................................
 
 .. image:: two_sided.png
    :scale: 50
    :align: right
 
-:func:`scipy.stats.ttest_1samp` tests if the population mean of data is
-likely to be equal to a given value (technically if observations are
-drawn from a Gaussian distributions of given population mean). It returns
+:func:`scipy.stats.ttest_1samp` tests the null hypothesis that the mean
+of the population underlying the data is equal to a given value. It returns
 the `T statistic <https://en.wikipedia.org/wiki/Student%27s_t-test>`_,
 and the `p-value <https://en.wikipedia.org/wiki/P-value>`_ (see the
 function's help)::
 
     >>> sp.stats.ttest_1samp(data['VIQ'], 0)   # doctest: +ELLIPSIS
     TtestResult(statistic=30.088099970..., pvalue=1.32891964...e-28, df=39)
 
-.. tip::
+The p-value of :math:`10^-28` indicates that such an extreme value of the statistic
+is unlikely to be observed under the null hypothesis. This may be taken as
+evidence that the null hypothesis is false and that the population mean IQ
+(VIQ measure) is not 0.
+
+Technically, the p-value of the t-test is derived under the assumption that
+the means of samples drawn from the population are normally distributed.
+This condition is exactly satisfied when the population itself is normally
+distributed; however, due to the central limit theorem, the condition is
```

Comment:
I've seen this written before, but it seems to me that it is safest to say that the sample t-statistic is distributed according to the t-distribution. I'm not sure whether this is guaranteed under the assumption stated above.